bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -676,38 +686,45 @@ public void dispose() {
         if (hueBridge != null) {
             hueBridge = null;
         }
+        ServiceRegistration<?> localServiceRegistration = serviceRegistration;
+        if (localServiceRegistration != null) {
+            // remove trustmanager service
+            localServiceRegistration.unregister();
+            serviceRegistration = null;
+        }
     }
 
     @Override
     public void initialize() {
-        logger.debug("Initializing hue bridge handler.");
+        logger.debug("Initializing Hue Bridge handler ...");
         hueBridgeConfig = getConfigAs(HueBridgeConfig.class);
 
-        String ip = hueBridgeConfig.getIpAddress();
+        String ip = hueBridgeConfig.ipAddress;
         if (ip == null || ip.isEmpty()) {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
                     "@text/offline.conf-error-no-ip-address");
         } else {
             if (hueBridge == null) {
-                hueBridge = new HueBridge(ip, hueBridgeConfig.getPort(), hueBridgeConfig.getProtocol(), scheduler);
-                hueBridge.setTimeout(5000);
+                if (HueBridgeConfig.HTTPS.equals(hueBridgeConfig.protocol)) {
+                    // register trustmanager service
+                    HueTlsTrustManagerProvider tlsTrustManagerProvider = new HueTlsTrustManagerProvider(
+                            ip + ":" + hueBridgeConfig.getPort(), hueBridgeConfig.useSelfSignedCertificate);
+                    serviceRegistration = FrameworkUtil.getBundle(getClass()).getBundleContext()
+                            .registerService(TlsTrustManagerProvider.class.getName(), tlsTrustManagerProvider, null);
+                }
+
+                hueBridge = new HueBridge(httpClient, ip, hueBridgeConfig.getPort(), hueBridgeConfig.protocol,
+                        scheduler);
 
                 updateStatus(ThingStatus.UNKNOWN);
 
-                // Try a first connection that will fail, then try to authenticate,
-                // and finally change the bridge status to ONLINE
-                initJob = scheduler.submit(new PollingRunnable() {
-                    @Override
-                    protected void doConnectedRun() throws IOException, ApiException {
-                    }
-                });
+                onUpdate();
             }
-            onUpdate();
```

Comment:
I am not sure in which case initialize could be called while hueBridge is not null, but with your changes the different polling will no more be started in this case.